addons/mail/static/src/js/models/threads/mixins/thread_typing_mixin.js

```diff
@@ -209,6 +213,9 @@ var ThreadTypingMixin = {
     unregisterTyping: function (params) {
         var partnerID = params.partnerID;
         this._othersTypingTimers.unregisterTimer({ timerID: partnerID });
+        if (!_.contains(this._typingPartnerIDs, partnerID)) {
```

Comment:
I would try to find a better commit message, this one seems confusing, as if we would slow down something.